singer/pom.xml

```diff
@@ -189,6 +189,12 @@
             <artifactId>aws-java-sdk-s3</artifactId>
             <version>1.11.35</version>
         </dependency>
+        <dependency>
+		    <groupId>com.salesforce.kafka.test</groupId>
```

Comment:
Is this a reliable dependency? The work `test` in the group id makes me wonder.

Reply:
Yes, it's called test because it's a package used for testing.